_rules/element-lang-valid-de46e4.md

```diff
@@ -27,9 +27,9 @@ acknowledgments:
 
 ## Applicability
 
-This rules applies to any HTML element with a `lang` [attribute value][] that is not empty (`""`) and for which all of the following is true:
+This rule applies to any HTML element with a `lang` [attribute value][] that is not empty (`""`) and for which all of the following is true:
 
-- the element is a [descendant][] in the [flat tree][] of a `body` element; and
+- the element is an [inclusive descendant][] in the [flat tree][] of a `body` element; and
```

Comment:
I think it is OK for this rule to check `lang` on a `body` element. Added it to match the Applicability of the new rule.